Networking/NetworkingTests/Remote/SitePluginsRemoteTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+@testable import Networking
+
+/// SitePluginsRemote Unit Tests
+///
+class SitePluginsRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    /// Dummy Site ID
+    ///
+    let sampleSiteID: Int64 = 1234
+
+    /// Repeat always!
+    ///
+    override func setUp() {
+        network.removeAllSimulatedResponses()
+    }
+
+    // MARK: - Load plugins tests
+
+    /// Verifies that loadPlugins properly parses the sample response.
+    ///
+    func testLoadPluginsProperlyReturnsPlugins() {
+        let remote = SitePluginsRemote(network: network)
+        let expectation = self.expectation(description: "Load site plugins")
+
+        network.simulateResponse(requestUrlSuffix: "plugins", filename: "plugins")
+        remote.loadPlugins(for: sampleSiteID) { (sitePlugins, error) in
+            XCTAssertNil(error)
+            XCTAssertNotNil(sitePlugins)
+            XCTAssertEqual(sitePlugins?.count, 5)
+            expectation.fulfill()
+        }
+
+        wait(for: [expectation], timeout: Constants.expectationTimeout)
+    }
+
+    /// Verifies that loadGeneralSettings properly relays Networking Layer errors.
+    ///
+    func testLoadPluginsProperlyRelaysNetwokingErrors() {
```

Comment:
```suggestion
    func test_loadPlugins_properly_relays_netwoking_errors() {
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/8e320fa88e27122e36d57a3de6e396bc6ec0ce37